tests/CoreFX/runtest/CoreFXTestHarness/Test.csproj

```diff
@@ -0,0 +1,67 @@
+<Project DefaultTargets="LinkNative">
+    <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+    <Import Project="$(TestRootDir)\dependencies.props" />
+
+    <!-- Override Default MSBuild properties-->
+    <PropertyGroup>
+        <TargetName>$(ExecutableName)</TargetName>
+        <TargetExt>.exe</TargetExt>
+        <OutputType>Exe</OutputType>
+    </PropertyGroup>
+    
+    <PropertyGroup>
+        <!-- TODO - need a general way to specify version without relying on the test directory structure being intact, i.e. import dependencies from \tests\ -->
+        <ToolsDir>$(FrameworkLibPath)\..\tools\</ToolsDir>
+        <TargetFramework>netcoreapp2.0</TargetFramework>
+        <!-- Don't warn if some dependencies were rolled forward -->
+        <NoWarn>$(NoWarn);NU1603</NoWarn>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <RdXmlFile Include="default.rd.xml" />
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="xunit.abstractions">
+            <Version>$(XunitAbstractionsVersion)</Version>
+        </PackageReference>
+        <PackageReference Include="xunit.assert">
+            <Version>$(XunitPackageVersion)</Version>
+        </PackageReference>
+        <PackageReference Include="xunit.extensibility.core">
+            <Version>$(XunitPackageVersion)</Version>
+        </PackageReference>
+        <PackageReference Include="xunit.core">
+            <Version>$(XunitPackageVersion)</Version>
+        </PackageReference>
+        <PackageReference Include="xunit.runner.utility">
+            <Version>$(XunitPackageVersion)</Version>
+        </PackageReference>
+        <PackageReference Include="xunit.extensibility.execution">
+            <Version>$(XunitPackageVersion)</Version>
+        </PackageReference>
+        <PackageReference Include="microsoft.xunit.netcore.extensions">
+            <Version>$(XunitNetcoreExtensionsVersion)</Version>
+        </PackageReference>
+    </ItemGroup>
+    <ItemGroup>
+        <!-- Some tests consist of multiple assemblies - make sure ILC sees them -->
+        <IlcCompileInput Include="$(MSBuildProjectDirectory)\*.dll" />
+        <!-- TODO Get both assemblies from somewhere else?  -->
```

Comment:
It should be included here https://github.com/dotnet/corert/blob/master/src/Framework/Framework-uapaot.depproj#L26; or stop being excluded here https://github.com/dotnet/corert/blob/master/src/Framework/Framework.depproj#L42 .